OpenRA.Editor/Form1.cs

```diff
@@ -238,7 +238,9 @@ void PrepareMapResources(ModData modData, Map map)
 					if (rsi != null && rsi.EditorPalette != null && rsi.EditorPalette.Contains("terrain"))
 						templatePalette = palette;
 
-					var template = RenderUtils.RenderActor(info, tileset, templatePalette);
+					var race = Program.Rules.Actors["world"].Traits.WithInterface<CountryInfo>().First().Race;
```

Comment:
It means that if you merge both this and #4496 into a common test branch you'll find that `RenderEditorOnly` generates invalid renderables that crash the renderer.  After working around that (for testing purposes I just disabled its rendering completely) everything else just works wrt owner and race artwork.